examples/convert_eeg_to_bids.py

```diff
@@ -26,7 +26,8 @@
 import mne
 from mne.datasets import eegbci
 
-from mne_bids import write_raw_bids, make_bids_basename
+from mne_bids import write_raw_bids
+from mne_bids.write import make_bids_basename
```

Comment:
make_bids_basename is also used in the reading

Reply:
I didn't find it in `read.py` :thinking: could you point me to the spot?

Also, is this a call for action or a statement?

Reply:
sorry, what I meant is that it is used in tandem with the read [example](https://github.com/mne-tools/mne-bids/blob/master/examples/convert_mne_sample.py#L68).